From 25c8d00cf4d0ad9e464c44ae8564f8f78f8da119 Mon Sep 17 00:00:00 2001
From: Amit Shah <amit.shah@redhat.com>
Date: Fri, 11 Jul 2014 10:09:03 -0500
Subject: [CHANGE 07/29] tests: vmstate static checker: version mismatch inside
 a Description
To: rhvirt-patches@redhat.com,
    jen@redhat.com

RH-Author: Amit Shah <amit.shah@redhat.com>
Message-id: <42e1d8959484f1d5c4e085ec4b9516aba940c105.1405072585.git.amit.shah@redhat.com>
Patchwork-id: 59787
O-Subject: [RHEL7.1 qemu-kvm PATCH 06/18] tests: vmstate static checker: version mismatch inside a Description
Bugzilla: 1118707
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Dr. David Alan Gilbert (git) <dgilbert@redhat.com>

Signed-off-by: Amit Shah <amit.shah@redhat.com>
Acked-by: Juan Quintela <quintela@redhat.com>
Signed-off-by: Juan Quintela <quintela@redhat.com>
(cherry picked from commit 4efa6e1d6473dbd775a5dd7eb976fa11501a94d2)
Signed-off-by: Amit Shah <amit.shah@redhat.com>
Signed-off-by: jen <jen@redhat.com>
---
 tests/vmstate-static-checker-data/dump2.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/vmstate-static-checker-data/dump2.json b/tests/vmstate-static-checker-data/dump2.json
index f405534..36a9b4b 100644
--- a/tests/vmstate-static-checker-data/dump2.json
+++ b/tests/vmstate-static-checker-data/dump2.json
@@ -829,7 +829,7 @@
     "minimum_version_id": 2,
     "Description": {
       "name": "fdc",
-      "version_id": 2,
+      "version_id": 1,
       "minimum_version_id": 2,
       "Fields": [
         {
-- 
1.9.3

